change/@fluentui-react-toast-27a36992-a9ec-40c0-9e5c-b9a7538c7193.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30715&commitId=2443c94489deb714f80d87f8bde0be54819fc23f&env=prod&clientType=FLUENTUI)


  <details><summary><b>Toast</b> 15 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Toast.Full Toast - Dark Mode.Toast visible.chromium.png|52| Changed|
|Toast.Full Toast.Toast visible.chromium.png|14| Changed|
|Toast.Full Toast Inverted - Dark Mode.Toast visible.chromium.png|17| Changed|
|Toast.Title Only - Dark Mode.Toast visible.chromium.png|52| Changed|
|Toast.Full Toast Inverted.Toast visible.chromium.png|32| Changed|
|Toast.Full Toast - RTL.Toast visible.chromium.png|14| Changed|
|Toast.Title Only - RTL.Toast visible.chromium.png|14| Changed|
|Toast.Title Only Inverted - Dark Mode.Toast visible.chromium.png|17| Changed|
|Toast.Title Only Inverted.Toast visible.chromium.png|32| Changed|
|Toast.Without Subtitle - Dark Mode.Toast visible.chromium.png|52| Changed|
|Toast.Title Only.Toast visible.chromium.png|14| Changed|
|Toast.Without Subtitle - RTL.Toast visible.chromium.png|14| Changed|
|Toast.Without Subtitle Inverted - Dark Mode.Toast visible.chromium.png|17| Changed|
|Toast.Without Subtitle Inverted.Toast visible.chromium.png|32| Changed|
|Toast.Without Subtitle.Toast visible.chromium.png|14| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>